### PR TITLE
Cu 33yqkug portfolio page henry quillin

### PR DESCRIPTION
**Pull request by HenryQuillin**
### What 
Creates Wallet page and Portfolio (royalties list)
### Also 
Makes FilledButton more reusable
### Demo 
Screen Shot 2022-07-26 at 11 10 43 AM
Screen Shot 2022-07-26 at 11 12 14 AM

### CODE REVIEW

**Comment by j-mateo** [comment]
Task linked: [CU-33yqkug Wallet Page](https://app.clickup.com/t/33yqkug)

File: src/components/TableDropdownSelect.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      color: theme.colors.white,
+      transform: "scale(1.2)",
+      paddingBottom: "5px",
+    },
+    "& .MuiSelect-iconOpen": {
+      color: theme.colors.white,
+      transform: "rotate(180deg)",
+    },
+    "& .css-6t9ju4-MuiSelect-select-MuiInputBase-input-MuiOutlinedInput-input.css-6t9ju4-MuiSelect-select-MuiInputBase-input-MuiOutlinedInput-input.css-6t9ju4-MuiSelect-select-MuiInputBase-input-MuiOutlinedInput-input":
+      {},
```
Comment (escobarjonatan) [inline src/components/TableDropdownSelect.tsx:22]: *Feedback requested comment*

was this `{}` left on by accident? also, can this selector be reduced or is the specificity required in this case?

File: src/pages/home/wallet/TabbedContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+          } }
+          value={ value }
+          onChange={ handleChange }
+        >
+          <StyledTab label={ label1 } { ...a11yProps(0) } />
+          <StyledTab label={ label2 } { ...a11yProps(1) } />
+        </Tabs>
+      </Box>
+      <TabPanel value={ value } index={ 0 }>
+        <Component1 />
```
Comment (escobarjonatan) [inline src/pages/home/wallet/TabbedContainer.tsx:92]: Do you think we can be more specific to the naming of these two components: Component1/Component2? The answer might be no but just want to make sure

File: src/pages/home/wallet/UnclaimedRoyalties.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+export const UnclaimedRoyalties = ({
+  unclaimedRoyalties,
+}: UnclaimedRoyaltiesProps) => {
+  return (
+    <Box
+      sx={ {
+        backgroundColor: theme.colors.grey600,
+        padding: 2.5,
+        maxWidth: "400px",
+        height: "100px",
```
Comment (escobarjonatan) [inline src/pages/home/wallet/UnclaimedRoyalties.tsx:18]: I wonder if we should be setting a specific height here, i am afraid there could be an issue with the inner content being bigger than 100px

File: src/pages/home/wallet/UnclaimedRoyalties.tsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+            } }
+          >
+            ROYALTIES
+          </Box>
+        </Typography>
+        <Typography fontSize="28px" fontWeight={ 700 }>
+          ${ unclaimedRoyalties }
+        </Typography>
+      </Box>
+      <Button
```
Comment (escobarjonatan) [inline src/pages/home/wallet/UnclaimedRoyalties.tsx:47]: *Feedback requested comment*

Should we be using one of our button variants instead or maybe add it as an option if we plan to reuse this button style?

File: src/components/TableDropdownSelect.tsx (hunk 1 above)
Comment (HenryQuillin) [reply src/components/TableDropdownSelect.tsx]: Actually can't figure out why I added that selector. Removing it now

File: src/pages/home/wallet/TabbedContainer.tsx (hunk 2 above)
Comment (HenryQuillin) [reply src/pages/home/wallet/TabbedContainer.tsx:92]: Maybe LeftTabComponent / RightTabComponent?

File: src/pages/home/wallet/UnclaimedRoyalties.tsx (hunk 3 above)
Comment (HenryQuillin) [reply src/pages/home/wallet/UnclaimedRoyalties.tsx:18]: Changed to minHeight

File: src/pages/home/wallet/UnclaimedRoyalties.tsx (hunk 4 above)
Comment (HenryQuillin) [reply src/pages/home/wallet/UnclaimedRoyalties.tsx]: Good call. Created a new component in elements.

File: src/pages/home/wallet/TabbedContainer.tsx (hunk 2 above)
Comment (escobarjonatan) [reply src/pages/home/wallet/TabbedContainer.tsx:92]: Sounds good to me 👍🏽

**Review by escobarjonatan — APPROVED** [review]

**Pull request merged**